src/katgpucbf/utils.py

```diff
@@ -20,14 +20,19 @@
 import logging
 import signal
 from asyncio import get_event_loop
+from enum import Enum
```

Comment:
I remember you lobbied for `IntEnum` in `aiokatcp`, any reason you're using the 'normal' one here?

Reply:
I needed the one in aiokatcp to be an IntEnum because I needed to use `max()` with it, which relies on comparison operators.

The one I'm using here takes the place of values, not statuses. So I don't need an IntEnum.